src/core/libraries/np_trophy/np_trophy.cpp

```diff
@@ -998,14 +975,14 @@ int PS4_SYSV_ABI sceNpTrophyUnlockTrophy(OrbisNpTrophyContext context, OrbisNpTr
                                 .set_value(std::to_string(trophyTimestamp.tick).c_str());
                         }
 
-                        g_trophy_ui.AddTrophyToQueue(trophyId, currentTrophyName);
+                        g_trophy_ui.AddTrophyToQueue(currentIconPath, currentTrophyName);
                     }
                 }
             }
         }
 
         if (std::string(platinumIt->attribute("unlockstate").value()).empty()) {
```

Comment:
```suggestion
        if (platinumIt->attribute("unlockstate").empty()) {
```
Is this a valid transformation? If not consider using std::string_view instead. And possibly a helper lambda to reduce some code duplication